jenkins/pipelines/cd/optimization-build-tidb-linux-amd.groovy

```diff
@@ -115,8 +116,7 @@ try {
                         }
                     }
                     container("golang") {
```

Comment:
Do we need to remove container("golang")

Reply:
Nodes can be defined in a pipeline and then used, however, default execution always goes to the jnlp container. Specify the container we want to execute  task in is necessary and reasonable.